src/plugins/map/MapObject.cpp

```diff
@@ -1051,6 +1052,19 @@ void UpdateDefaultMapLocParams()
 	gDefaultMapLocParams.circleColor = MQColor(r, g, b);
 }
 
+void ResetMapLocOverrides()
+{
+	gOverrideMapLocParams.lineSize = gDefaultMapLocParams.lineSize;
+	gOverrideMapLocParams.width = gDefaultMapLocParams.width;
+	gOverrideMapLocParams.color.Red = gDefaultMapLocParams.color.Red;
+	gOverrideMapLocParams.color.Green = gDefaultMapLocParams.color.Green;
+	gOverrideMapLocParams.color.Blue = gDefaultMapLocParams.color.Blue;
+	gOverrideMapLocParams.circleRadius = gDefaultMapLocParams.circleRadius;
+	gOverrideMapLocParams.circleColor.Red = gDefaultMapLocParams.circleColor.Red;
+	gOverrideMapLocParams.circleColor.Green = gDefaultMapLocParams.circleColor.Green;
+	gOverrideMapLocParams.circleColor.Blue = gDefaultMapLocParams.circleColor.Blue;
```

Comment:
`gOverrideMapLocParams = gDefaultMapLocParams`

Reply:
won't that make them both point to the same instance instead of altering the values of one instance?

Reply:
no, they are value types, not references.